File: modulo1/segmento3/apunte5For/ejercicio6.py
linea=""
vueltasi= int(input("Ingrese el maximo"))

for i in range(vueltasi):
	linea = linea + "x"
	print(linea)

for i in reversed(range(vueltasi - 1)):
	linea=""
	for j in range(i):
		linea = linea + "x"

	print(linea)

File: modulo1/segmento3/apunte5For/Ejercicio6.java
import java.util.Scanner;

public class Ejercicio6{
	public static void main(String args[]){
		Scanner teclado = new Scanner(System.in);
		
		System.out.println("Ingrese el maximo");
		Integer max= Integer.parseInt(teclado.nextLine());

		String linea= "";

		for (Integer i=0;i<max; i++){
			linea+= "x";
			System.out.println(linea);
		}

		for (Integer i=max-1;i>0;i--){
			linea="";
			for(Integer j=0;j<i;j++){
				linea+="x";
			}
			System.out.println(linea);
		
		}
	
	
	}


}
